### PR TITLE
Decompose into private modules

### PR DESCRIPTION
**Pull request by necolas**
cc @brettstimmerman

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,235 @@
...
+}
+
+/**
+ * BiDi flip the given `declaration`.
+ *
+ * @param {Object} declaration
+ */
+
+function flipDeclaration(declaration) {
+  var name = (declaration.property || declaration.name).toLowerCase();
```
Comment (necolas) [inline index.js:109]: btw, what is the `declaration.name` guarding against?

Comment (brettstimmerman) [inline index.js:109]: Almost positive `declaration.name` can be removed.

**Comment by brettstimmerman** [comment]
Excellent! :ship: ship it

**Comment by necolas** [comment]
Applied in 67913145bde5f7976711bc998e1299778a26502a

**Pull request closed without merging**